cli.js

```diff
@@ -47,7 +56,7 @@ process.stdin.on('keypress', (ch, key) => {
 
 	if (key.name === 'escape' || (key.ctrl && key.name === 'c')) {
 		if (query.length <= 1) {
-			logUpdate();
+			logUpdate(`${pre}${chalk.dim('Hey! You were supposed to use me')}\n\n`);
```

Comment:
@CodeDotJS People might clear out the text before exciting. I don't really see any gain of printing this.